src/suites/rulesets/lighthouse-api-standards.yaml

```diff
@@ -34,6 +34,16 @@ rules:
       functionOptions:
         match: /^[A-Z]+[A-Z0-9]+(?:_[A-Z0-9]+)*$/
 
+  va-property-date-format:
+    description: Dates must follow the format YYYY-MM-DD or YYYY-MM
+    message: Dates must follow the format YYYY-MM-DD or YYYY-MM
+    severity: error
+    given: $..properties.[?(@.format == "date")].example
+    then:
+      function: pattern
+      functionOptions:
+        match: /([12]\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))$|([12]\d{3}-(0[1-9]|1[0-2]))$/
```

Comment:
`[12]\d{3}` What happens in the year 3000? 😀 I'm sure it will be out of commission by then but you could also do `\d{4}`